app/views/home/businesses.html.erb

```diff
@@ -35,14 +35,14 @@
 
   <div class="col-sm-6 col-md-12 float-left float-md-none pl-sm-4 pl-md-0 clearfix">
     <div class="col-12 col-md-6 float-md-right pl-0 pl-md-3 mb-3 mb-sm-0">
-      <h3 class="alt-h3 mb-1">Improve your technology</h3>
+      <h3 class="alt-h3 mb-1">Reduce overhead</h3>
```

Comment:
@yourfrienderin asks: _Is this more of a "proactive investment" than an overhead reduction? I ask beause you're essentially trading off time doing one for time doing the other._

@mikemcquaid I see her point. Also, custom forks are maintained for a reason (ex. specific use case) and I don't think we want to suggest that's categorically bad.

Ideas for clarifying how this is a timesaving investment?